client/components/site-title/index.jsx

```diff
@@ -45,17 +45,19 @@ class SiteTitleControl extends React.Component {
 				<FormFieldset>
 					<FormLabel htmlFor="blogname">{ this.props.translate( 'Site Title' ) }</FormLabel>
 					<FormTextInput
+						autoFocus
```

Comment:
I'd prefer making that be set from an optional prop that defaults to `false`. Main rationale is avoiding the auto focusing on unexpected places (like http://calypso.localhost:3000/devdocs/design for example).

Reply:
Okay, will do. A bit specific/not too elegant (`autoFocusBlogname`?), which is why I avoided it in the first place, but you're right.

Reply:
Just `autoFocus` will be enough I think. Wanting to autofocus the tagline field would be too edgy anyway 😉

Reply:
> Just `autoFocus` will be enough I think.

Not descriptive enough for me, TBH -- if I was seeing that, I'd be wondering if the entire component would somehow be highlighted or whatever else that is supposed to mean...